_posts/2019/2019-05-07-react-hooks-what-all-the-fuss-is-about.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: React Hooks - What all the fuss is about
+date: 2019-05-07
+header_image: public/react-hooks.jpg
+header_position: center
+category: coding
```

Comment:
After having had a look at the second post, I am wondering if this fits better into "tech-stories"?

Reply:
Sure, after all, it's not very technical.